corehq/apps/geospatial/static/geospatial/js/geo_config.js

```diff
@@ -5,20 +5,22 @@ hqDefine("geospatial/js/geo_config", [
 ], function (
     $,
     ko,
-    initialPageData,
+    initialPageData
 ) {
-    const CUSTOM_USER_PROP = "custom_user_property";
-
     var geoConfigViewModel = function (configData) {
         'use strict';
         var self = {};
 
-        var data = configData.get('config')
+        const gpsCaseProps = configData.get('gps_case_props');
+        self.geoCasePropOptions = ko.observableArray(Object.keys(gpsCaseProps));
```

Comment:
This is an observable since we are using the `data-bind` attribute in `forms.py` to set the select options. We could alternatively use JQuery to set the options of the select, but this method felt cleaner.

Reply:
Oh, I thought we could simply bind standard properties.
Yes yes, using knockoutJS would be preferred over JQuery.